app/components/ChilDevelopmentCollapsibleItem.tsx

```diff
@@ -30,9 +30,17 @@ import { addSpaceToHtml, removeParams } from '../services/Utils';
 import { PopupCloseVideo, PopupCloseContainer } from '@components/shared/ModalPopupStyle';
```

Comment:
File was created earlier but name has a spelling issue. Let's please correct it if impact is small otherwise we can ignore it.

Reply:
Noted and will do it